src/JsonSchema/Constraints/ConstraintInterface.php

```diff
@@ -59,5 +59,10 @@ public function isValid();
      * @param mixed            $i
      * @throws \JsonSchema\Exception\ExceptionInterface
      */
+    public function _check(&$value, $schema = null, JsonPointer $path = null, $i = null);
```

Comment:
I think we may have different ideas on what constitutes acceptable grounds for breaking a public API.

My position on this kind of thing is that breaking a five-year-old public API immediately and without warning when a couple of aliases can allow for graceful deprecation over the next few releases is unconscionable, regardless of how trivial the fix on their end. Users deserve more respect than that.

That said, this is not my library... so if the consensus is that breaking the API is in fact the desired outcome, then I'll go along with that. But I'm not happy about it.

I wrote this PR because I thought that your suggestion was to deprecate `coerce`, unify behavior with `check`, and expose `checkMode` in the public API - and this PR seemed like the best way to go about it - but have I misunderstood your original intention?